### DISCUSSION TITLE
Welcome to safetyGraphics Discussions!

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/safetyGraphics#454] [General]
## 👋 Welcome!

GitHub just moved their discussion board feature out of beta, and I'd like to try it out here in the safetyGraphics package.  The tool is suitable for lots of general tasks - GitHub suggests a few including: 

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I'd also like our group to think about using this discussion board for the following: 

  * Post meeting minutes from our ISG workforce full team and technical meetings.
  * Ask technical questions about the safetyGraphics platform (these might turn in to issues!).
  * Gather general feedback on safetyGraphics - especially from alpha and beta testing.
 
I've also created a [discussion board in our safetyCharts repo](https://github.com/SafetyGraphics/safetyCharts/discussions), where I think we might discuss new charts that we want to create. 

Anyone have other ideas about how we might use this tool??

**Comment by xni7** [comment]
Seems very nice. some quick thoughts

- Agreed on the second bullet above. This could be a place to formulate and triage questions vs. issues. Not all questions result into issues, nor are all findings bugs.
- Should consider adding a "Meeting" minutes category to allow filtering to meetings minutes vs. other discussions.
- It's worth discussing what are the "categories" meta data to better organize the discussions. The search feature is nice
- Let's discuss the use cases of this vs. Slack and align on the usage of both tools

**Reply by jwildfire** [reply]
Just added the meetings tag. Good idea!
